sklearn/metrics/pairwise.py

```diff
@@ -253,6 +253,7 @@ def euclidean_distances(
 
     Y : {array-like, sparse matrix} of shape (n_samples_Y, n_features), \
             default=None
+        If `None`, uses Y=X.
```

Comment:
Could you put some backticks around Y=X here and elsewhere

```suggestion
        If `None`, uses `Y=X`.
```

Reply:
yes, no problem! I just did @glemaitre